SwiftExample/SwiftExample_tvOS/AppDelegate.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AppDelegate.swift
+//  SwiftExample_tvOS
```

Comment:
Is it possible to use the existing example for `tvOS`? Would a bit cleaner and less files in the project